src/main/flight/pid_luxfloat.c

```diff
@@ -85,8 +85,13 @@ STATIC_UNIT_TESTED int16_t pidLuxFloatCore(int axis, const pidProfile_t *pidProf
     // -----calculate P component
     float PTerm = luxPTermScale * rateError * pidProfile->P8[axis] * PIDweight[axis] / 100;
     // Constrain YAW by yaw_p_limit value if not servo driven, in that case servolimits apply
-    if (axis == YAW && pidProfile->yaw_p_limit && motorCount >= 4) {
-        PTerm = constrainf(PTerm, -pidProfile->yaw_p_limit, pidProfile->yaw_p_limit);
+    if (axis == YAW) {
+        if (pidProfile->yaw_lpf) {
+            PTerm = filterApplyPt1(PTerm, &yawFilterState, pidProfile->yaw_lpf, dT);
+        }
+        if (pidProfile->yaw_p_limit && motorCount >= 4) {
```

Comment:
@hydra , if there are fewer than 4 motors then yaw is handled by a servo, and so there is no need to contrain the yaw.

The reason the yaw is constrained is so that too much motor rpm is not used for yaw, leaving nothing left over for the other axes. This yaw contraint makes a quadcopter much more resilient to minor crashes (eg clipping a gate, or glancing off a wall).